docs/guide/machine-learning.asciidoc

```diff
@@ -88,6 +88,37 @@ Downloading: 100%|██████████| 249M/249M [00:23<00:00, 11.2MB
 100%|██████████| 63/63 [00:12<00:00,  5.02it/s]
 ------------------------
 
+[discrete]
+[[ml-nlp-pytorch-docker]]
+==== Import model with Docker
+
+IMPORTANT: To use the docker container, you will need to clone the eland repository: https://github.com/elastic/eland
+
+If you want to use Eland without installing it, in order to just run the available scripts, can build the Docker
+container:
```

Comment:
It might be a bit easier to read this way.
```suggestion
If you want to use Eland without installing it, you can build the Docker container to run the available scripts:
```